pkgs/by-name/ch/chirpstack-concentratord/package.nix

```diff
@@ -1,8 +1,9 @@
-{ lib
```

Comment:
Please squash this commit into the init commits

Reply:
Done 👍🏻